crypto/README.md

```diff
@@ -34,16 +44,34 @@ The parameters used to instantiate the functions are:
   - S-Box degree: 3.
   - Target security level: 124-bits.
 
+### Griffin hash function implementation
+Griffin hash function is implemented according to the Griffin [specifications](https://eprint.iacr.org/2022/403.pdf) with the following differences:
+* We set the number of rounds to 7, which implies a 15% security margin instead of the 20% margin used in the specifications (a 20% margin rounds up to 8 rounds), similarly to the motivation on Rescue above.
+* When hashing a sequence of elements, we do not append Fp(1) followed by Fp(0) elements to the end of the sequence as padding. Instead, we initialize one of the capacity elements to the number of elements to be hashed, and pad the sequence with Fp(0) elements only. However, this also means that our instantiation of Griffin cannot be used in a stream mode as the number of elements to be hashed must be known upfront.
```

Comment:
There are actually some concerns about potential security degradation from using this padding rule. It might be better to use a padding rule similar to the one used in RPO.

Reply:
The Hirose padding? Sounds fair, we've used this also in Anemoi.
Should I wait until further discussion is made or update it right away?

Reply:
The Hirose padding? Sounds fair, we've used this also in Anemoi.
Should I wait until further discussion is made or update it right away?